src/infra/config/dex/balancer/file.rs

```diff
@@ -43,6 +53,8 @@ pub async fn load(path: &Path) -> super::Config {
                 .unwrap_or(contracts.balancer_vault),
             settlement: base.contracts.settlement,
             block_stream: base.block_stream.clone(),
+            chain_id: config.chain_id,
+            query_batch_swap: config.query_batch_swap.unwrap_or(false),
```

Comment:
Disabled by default, so cached values will be used to find an optimal route leading to not a 100% return amount accuracy. We can play around with this config to understand if it affects the latency.